logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepository.kt

```diff
@@ -89,17 +89,15 @@ class MessageDataSource(
     override suspend fun softDeleteMessage(messageUuid: String, conversationId: ConversationId): Either<CoreFailure, Unit> {
         messageDAO.updateMessageVisibility(
             visibility = MessageEntity.Visibility.DELETED,
-            conversationId = idMapper.toDaoModel(conversationId),
             id = messageUuid
         )
         //TODO: Handle failures
         return Either.Right(Unit)
     }
 
-    override suspend fun hideMessage(messageUuid: String, conversationId: ConversationId): Either<CoreFailure, Unit> {
+    override suspend fun hideMessage(messageUuid: String): Either<CoreFailure, Unit> {
```

Comment:
The message ID is not UNIQUE in the DB, so the conv ID is needed as well,
and do we need to actually delete the message from the DB instead of making it invisible?

Reply:
I get your point and agree with it. I'm working on the hide messages in other task [ the conversation id in hide messages is the self, so in this branch we can ignore that change will be fixed properly with this consideration in my other task]

Reply:
related ticket that I'm gonna fix that in: https://wearezeta.atlassian.net/browse/AR-1537

Reply:
if this change is needed for other feature, can we not merge it to dev but to another branch to avoid breaking dev and later merge the 2 together ?